### PR TITLE
support getting registered user details

### PR DESCRIPTION
**Pull request by xquanluu**
#263

### CODE REVIEW

**Review by davehorton — COMMENTED** [review]
does the swagger doc need to be updated?

**Review by davehorton — CHANGES_REQUESTED** [review]
shouldn't there have been an update to the swagger doc for this new API?

File: lib/routes/api/accounts.js
Comment (xquanluu) [reply lib/routes/api/accounts.js:168]: At the time created this PR, I would to avoid scan all keys if not required. If we want to get whole users, I may want to wait until @ajukes PR merged to redis helper lib

Comment (davehorton) [reply lib/routes/api/accounts.js:168]: Yes, but I would not want to change the behavior after we release this feature.  So I guess we will wait until I finish review and merge the other PR

Comment (xquanluu) [reply lib/routes/api/accounts.js]: I have created new PR for registra to support this feature after the other PR is already merged. Please release that to new version and I will update this PR after that soon.

Comment (davehorton) [reply lib/routes/api/accounts.js]: jambonz-mw-registrar@0.2.6 has that change

**Review by davehorton — APPROVED** [review]

**Pull request merged**
